scripts/bootstrap.sh

```diff
@@ -172,26 +172,7 @@ fi
 
 echo "Checking for .git file to identify local installation; \"fatal: not a git repository\" is expected elsewhere"
 git config --global --add safe.directory /var/www
-if [[ $(git rev-parse --is-inside-work-tree) ]]; then
-  # Find the php.ini file
-  PHP_INI=$(php -i | grep 'Loaded Configuration File' | awk '{print $NF}')
 
-  # Check if opcache is already disabled
-  if grep -q 'opcache\.enable\s*=\s*0' "$PHP_INI"; then
-    echo "OPCache is already disabled."
-  else
-    echo "Disabling OPCache..."
-    sed -i 's/^opcache\.enable\s*=.*/opcache.enable=0/' "$PHP_INI"
-    sed -i 's/^opcache\.enable_cli\s*=.*/opcache.enable_cli=0/' "$PHP_INI"
-    echo "OPCache disabled."
-  fi
-fi
-
-# php needs a restart so new relic ini changes take effect
```

Comment:
These lines (190-194 in original version) need to come back. They're about restarting php to load any changes that were made to /etc/php83/conf.d/newrelic.ini